docs/source/tutorials/datasets.rst

```diff
@@ -184,24 +184,26 @@ files:
     tokenizer = ...
     dataset = text_completion_dataset(
         tokenizer,
-        source="txt",
+        source="text",
         data_files="path/to/my_data.txt",
+        split="train",
     )
 
 .. code-block:: yaml
 
     # YAML config
     dataset:
       _component_: torchtune.datasets.text_completion_dataset
-      source: txt
+      source: text
       data_files: path/to/my_data.txt
+      split: train
 
 .. code-block:: bash
 
     # Command line
-    tune run full_finetune_single_device --config llama3/8B_full_single_device \
-    dataset=torchtune.datasets.text_completion_dataset dataset.source=txt \
-    dataset.data_files=path/to/my_data.txt
+    tune run full_finetune_distributed --config llama3/8B_full \
+    dataset=torchtune.datasets.text_completion_dataset dataset.source=text \
+    dataset.data_files=path/to/my_data.txt dataset.split=train
```

Comment:
Why this change? Shouldn't we have number of devices in here if we're doing this?

Reply:
single device recipe requires bitsandbytes, which needs to be installed separately by the user. I'd rather not use that as an example. But yes, I need to add number of devices..